### ISSUE TITLE
SES NoSuchMethodError: Class 'Destination' has no instance method 'toJson'.

### ISSUE THREAD

**Issue by latesr** [agilord/aws_client#172]
Whilst trying out your SES support, I encountered the following exception.

```Unhandled exception:
NoSuchMethodError: Class 'Destination' has no instance method 'toJson'.
Receiver: Instance of 'Destination'
Tried calling: toJson()
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:53:5)
#1      _flatten (package:shared_aws_api/src/protocol/query.dart:124:17)
#2      _SyncIterator.moveNext (dart:core-patch/core_patch.dart:146:12)
#3      _SyncIterator.moveNext (dart:core-patch/core_patch.dart:139:32)
#4      MapMixin.addEntries (dart:collection/maps.dart:172:23)
#5      new Map.fromEntries (dart:core/map.dart:187:17)
#6      flatQueryParams (package:shared_aws_api/src/protocol/query.dart:89:14)
#7      QueryProtocol._buildRequest (package:shared_aws_api/src/protocol/query.dart:74:26)
#8      QueryProtocol.send (package:shared_aws_api/src/protocol/query.dart:49:16)
#9      SES.sendEmail (package:aws_ses_api/email-2010-12-01.dart:2208:37)
```

The following code demonstrates this minimally:

```dart 
import 'package:aws_ses_api/email-2010-12-01.dart';

void main(List<String> arguments) async {
  final cred = AwsClientCredentials(
      accessKey: '',
      secretKey: '');
  final ses = SES(region: 'eu-west-1', credentials: cred);
  await ses.sendEmail(
      destination: Destination(toAddresses: ['dest@example.com']),
      message: Message(
        body: Body(
          text: Content(data: 'My test text'),
        ),
        subject: Content(data: 'Test subject'),
      ),
      source: 'source@example.com');
}
```

**Comment by Schwusch** [comment]
This can probably be solved at the same time as https://github.com/agilord/aws_client/issues/144. 

@isoos Are you doing any work in this?

**Comment by isoos** [comment]
> @isoos Are you doing any work in this?

Not at the moment :(

**Comment by Schwusch** [comment]
This should be resolved in [version 0.1.1](https://pub.dev/packages/aws_ses_api/versions/0.1.1). @latesr can you verify?

**Comment by Schwusch** [comment]
I'll close this for now. The activity indicates that this may not a problem anymore.
